lib/biran/configurinator.rb

```diff
@@ -19,9 +19,10 @@ def env= env
       end
 
       def env
-        return @end if @env
-        return Rails.env if defined? Rails
-        DEFAULT_ENV
+        return @env if @env
+        #return Rails.env if defined? Rails
+        #DEFAULT_ENV
+        @env = app_env
```

Comment:
This is the part I get confused and I know this isn't working but its illustrative of the value I want

Reply:
its included from config_defaults

Reply:
got it